File: java_backup/java.jimut/ICSE/HCF.java
import java.io.*;
class HCF
{
    public static void main(String args[])throws IOException
    {
        InputStreamReader ab = new InputStreamReader(System.in);
        BufferedReader cd = new BufferedReader(ab);
        System.out.println("Enter two numbers:");
        int n=Integer.parseInt(cd.readLine());
        int m=Integer.parseInt(cd.readLine());
        int p,i,s=1;
        p=(m*n)/2;
        for(i=1;i<=p;i++)
        {
            if(((m%i)==0)&&((n%i)==0))
            {
                s=i;
            }
        }
        System.out.println("HCF = "+s);
           
}
}

File: 6th_sem/MATLAB/CIA_IMP/Assignment 1/hcf.m
%function h=hcf(a,b) : Function to calculate hcf of 2 numbers
function h=hcf(a,b)
r=mod(a,b);
while r ~=0
    a=b;
    b=r;
    r=mod(a,b);
end
h=b;
end
